website/source/docs/vault-enterprise/hsm/index.html.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: "docs"
+page_title: "Vault Enterprise HSM Support"
+sidebar_current: "docs-vault-enterprise-hsm"
+description: |-
+  Vault Enterprise has HSM support, allowing for external master key storage and automatic unsealing.
+
+---
+
+# Vault Enterprise HSM Support
+
+HSM support is a feature of [Vault
+Enterprise](https://www.hashicorp.com/vault.html) that takes advantage of HSMs
+to provide two pieces of special functionality:
+
+ * Master Key Wrapping: Vault protects its master key by transiting it through
+   the HSM for encryption rather than splitting into key shares
+ * Automatic Unsealing: Vault stores its encrypted master key in storage,
+   allowing for automatic unsealing
+
+HSM support is currently limited to devices that support PKCS#11 interfaces and
+provide integration libraries. This includes AWS'
```

Comment:
I would also include Thales, Utimaco, SafeNet/Gemalto since those have all been tested. We don't want to give people the impression of only CloudHSM